NOnion/Exceptions.fs

```diff
@@ -2,13 +2,18 @@
 
 open System
 
+type NOnionException(msg: string, innerException: Exception) =
+    inherit Exception(msg, innerException)
+    new(msg) = NOnionException(msg, null)
```

Comment:
for the overload in which an innerException parameter is not used, you should not use null to call the base class, you should call the base class constructor that only has one parameter

Reply:
done